vms/platformvm/camino_service.go

```diff
@@ -542,6 +542,53 @@ func (s *CaminoService) Claim(_ *http.Request, args *ClaimArgs, reply *api.JSONT
 	return nil
 }
 
+type TransferArgs struct {
```

Comment:
We have seen in latest Banff, that avax starts removing all TX creating API functions (including keystore)
I personally would skip further tx relection in service -> don't touch service at all

Reply:
we'd need than some reliable fast way of creating txs locally, documented how-to, imo